### PR TITLE
Updates to the vignette

### LINKED ISSUE #155: User feedback from the test cycle before v 1.0
This issue is intended to track feedback from the test cycle which starts today. Users may submit comments and feedback in this thread or - if they prefer - via email and I will track it here.

The deadline for review is 28th of February 2020.

### PR DESCRIPTION
**Pull request by PatrickRWright**
These updates are in reference to the review feedback. I will also close the review feedback thread with this PR because I have tracked all other suggestions which I consider relevant as individual Issues.

closes #155

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -28,7 +28,7 @@ Description: This R package is intended to standardize and simplify working with
     exported from the clinical data management system (CDMS) secuTrial. The primary
     data export the package works with is a standard non-rectangular export.
 Depends:
-    R (>= 3.0.2),
+    R (>= 3.2),
```
Comment (aghaynes) [inline DESCRIPTION]: someone I asked to review it said that they needed to upgrade to 3.6 to install secuTrialR...?

Comment (PatrickRWright) [reply DESCRIPTION]: I was a bit surprised too that `haven 2.2.0` indicates `R >= 3.2` (I checked on their page). Should we just set `R >= 3.6` and be done with it?

Comment (aghaynes) [reply DESCRIPTION]: I've just installed it on 3.4.2... so I don't know what the other thing was about... I'll go speak to him

Comment (aghaynes) [reply DESCRIPTION]: he doesnt know... he had problems compiling with 3.5 apparently... he had to upgrade thinking that it was secutrialr... but maybe not.
We dont actually know if it works under 3.2 right...? maybe it's better to put a more recent version...

Comment (PatrickRWright) [reply DESCRIPTION]: What would you suggest?

Comment (aghaynes) [reply DESCRIPTION]: I dont know... there's still a lot of 3.5 users around...

Comment (PatrickRWright) [reply DESCRIPTION]: Lets stick with 3.2 as currently indicated then. At least according to the haven 2.2.0 docu this should work fine.

Comment (PatrickRWright) [reply DESCRIPTION]: If you agree with 3.2 you can merge this.

**Pull request closed without merging**